packages/webapp/src/containers/Task/TaskDate/index.jsx

```diff
@@ -10,11 +10,20 @@ function TaskDate({ history, match, location }) {
     history.back();
   };
   const isTransplantTask = useIsTaskType('TRANSPLANT_TASK');
+  const isMovementTask = useIsTaskType('MOVEMENT_TASK');
 
   const tasks = location.state.management_plan_id
     ? useSelector(tasksByManagementPlanIdSelector(location.state.management_plan_id))
     : [];
 
+  const getNextStepPath = () => {
+    return isTransplantTask
+      ? '/add_task/task_crops'
+      : isMovementTask
+        ? '/add_task/task_animal_selection'
+        : '/add_task/task_locations';
+  };
```

Comment:
One of the few cases I thinks chaining conditionals is readable -- but if not I can change.